12-say-grpc/backend/vendor/github.com/campoy/justforfunc/13-cgo-flite/flite/flite.go

```diff
@@ -0,0 +1,55 @@
+package flite
+
+// #cgo CFLAGS: -I /usr/include/flite
```

Comment:
Is there any easy way to make it work for all platforms?
The code is supposed to be cross platform according to the webpage
How would you test?